### PR TITLE
Some clarification in the `-h` documentation.

### PR DESCRIPTION
**Pull request by GerbrichFerdinands**
- updated documentation for `--config_file` 
- updated documentation for `--seed`

Hopefully solving #352 

@qubixes can you review? 
I needed >1 line for the updates, not sure if that's allowed and whether my use of `"""` is correct.

### CODE REVIEW

File: asreview/entry_points/base.py [hunk 1]
```diff
@@ -74,12 +74,15 @@ def _base_parser(prog=None, description=None):
...
+            hyperparameter values."""
     )
     parser.add_argument(
         "--seed",
         default=None,
         type=int,
-        help="Seed for models. Use integer between 0 and 2^32 - 1."
+        help="""Seed for the model (classifiers, balance 
+            strategies, feature extraction techniques, and query 
+            strategies). Use integer between 0 and 2^32 - 1."""
```
Comment (qubixes) [inline asreview/entry_points/base.py:86]: "Use integer" -> "Use an integer"

File: asreview/entry_points/base.py [hunk 2]
```diff
@@ -74,12 +74,15 @@ def _base_parser(prog=None, description=None):
         "--config_file",
         type=str,
         default=None,
-        help="Configuration file with model, balance, and query parameters"
+        help="""Configuration file with model parameters and
```
Comment (qubixes) [inline asreview/entry_points/base.py:77]: What is the difference between model parameters and hyperparameters?

**Comment by qubixes** [comment]
You can also use single quotes on multiple lines; it should merge them into a single string (since they are enclosed).

File: asreview/entry_points/base.py (hunk 2 above)
Comment (GerbrichFerdinands) [reply asreview/entry_points/base.py:77]: I think of it as: 

Model parameters are the global settings, like  `balance_strategy = double` 
Hyperparameter values are nested in the model parameters, like `alpha = 1.229374068496514`

```
[global_settings]
n_instances = 1
n_prior_included = 1
n_prior_excluded = 1
model = nb
balance_strategy = double
query_strategy = max
feature_extraction = tfidf

[balance_param]
a = 1.3783791240515941
alpha = 1.229374068496514
b = 0.9153629982662178

[feature_param]
ngram_max = 3
split_ta = 0
use_keywords = 1

[model_param]
alpha = 0.06586028856339346
```

**Comment by qubixes** [comment]
Ik bedoelde eigenlijk dit (voor voorbeelden):

https://github.com/qubixes/asreview-pro/blob/master/asreviewcontrib/pro/entrypoint.py

**Review by J535D165 — APPROVED** [review]

File: asreview/entry_points/base.py (hunk 2 above)
Comment (qubixes) [reply asreview/entry_points/base.py]: I think that might be confusing for the user. Perhaps we can call it simulation settings + model parameters?

Comment (GerbrichFerdinands) [reply asreview/entry_points/base.py]: I'm not a fan of simulation settings as these settings are not confined to simulations. 
How about model settings + model parameters?

Comment (qubixes) [reply asreview/entry_points/base.py]: Sure, that makes sense.

**Comment by J535D165** [comment]
Thanks!!

**Pull request merged**